public/pages/AnomalyCharts/containers/FeatureBreakDown.tsx

```diff
@@ -190,18 +190,22 @@ export const FeatureBreakDown = React.memo((props: FeatureBreakDownProps) => {
               }
               detectorEnabledTime={props.detector.enabledTime}
               titlePrefix={
-                props.selectedHeatmapCell
+                props.selectedHeatmapCell &&
+                props.title !== 'Sample feature breakdown'
```

Comment:
A lot of this is existing code, and there's other places where `Sample feature breakdown` as well as other chart titles aren't extracted into constants (see functions around [here](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/blob/main/public/pages/AnomalyCharts/utils/anomalyChartUtils.ts#L593)). I think it's a good idea. How about I create a separate issue to track this since it's not single-flow related?

Reply:
Created #55 to track